content/powershell/concepts/operators/operators.md

```diff
@@ -1,14 +1,122 @@
 ---
 Title: 'Operators'
-Description: ''
+Description: 'Operators are used to perform specific mathematical or logical functions on variables and values.'
 Subjects:
+  - 'Code Foundations'
   - 'Computer Science'
+  - 'PowerShell'
```

Comment:
```suggestion
  - 'Bash\Shell'
```
This is the subject as listed in documentation/subjects.md. You could add "PowerShell" to the subjects.md file, but it might appear redundant.